_posts/2015-02-17-aerogear-webpush.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: AeroGear WebPush
+---
+
+## AeroGear WebPush
+[AeroGear WebPush](https://github.com/aerogear/aerogear-webpush-server#aerogear-webpush-server-) is a proof of concept implementation of the 
+[WebPush Protocol](http://tools.ietf.org/html/draft-thomson-webpush-http2-02) specification with the purpose of gaining a better understanding of the specification. 
+
+As you might have guessed from the name this is about push notifications and is related to the [Push API](https://w3c.github.io/push-api/index.html). 
+In brief, it allows a device to maintain a single HTTP/2 connect which can service as many client applications as needed. For example, when used in combination with the Push API a service worker will handle the registration to the WebPush Server and manage the subscriptions for the applications. An application would in this case be tab in a web browser. 
```

Comment:
for _service worker_, should we link to ? http://www.html5rocks.com/en/tutorials/service-worker/introduction/